tests/Unit/PointwiseFunctions/GeneralRelativity/GrTestHelpers.cpp

```diff
@@ -0,0 +1,151 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/PointwiseFunctions/GeneralRelativity/GrTestHelpers.hpp"
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+
+template <typename DataType>
+Scalar<DataType> make_lapse(const DataType& structure) {
```

Comment:
I think calling this argument `datavector_for_size` as is done (almost everywhere) in the header file is clearer, and less likely to be confused with the Tensor structure.

This comment applies throughout this file and other files in this PR.

Reply:
I requested a move away from `datavector_for_size` because DataType is not necessarily DataVector.  I suggested "structure", but I'm not attached to the name.

Reply:
Fair enough. I would be okay with `structure`.

Alternatively, `placeholder_for_size` or `dummy_for_size` could work to indicate the intent without saying DataVector, though one could question whether they make sense if the DataType is a double...

Reply:
I've no strong feelings but I'm gonna go for `used_for_size` since I think it's the clearest